product-service/handlers/snsHelper.ts

```diff
@@ -0,0 +1,33 @@
+import { SNS } from "aws-sdk";
+
+export async function notifyProductInsertionSNS(
+  count: number,
+  totalPrice: number
+) {
+  const sns = new SNS();
+  return new Promise((resolve, reject) => {
+    const pricing = totalPrice > 10 ? "expensive" : "cheap";
+    sns.publish(
+      {
+        Subject: "New Products",
+        Message: `${count} products were successfully insereed into DB. Total Price: ${totalPrice}.`,
+        MessageAttributes: {
+          pricing: {
+            DataType: "String",
+            StringValue: pricing,
```

Comment:
Filter policy is based on this message attribute: 
- if the pricing (total price) is > 10 => send the message to the first subscriber
- otherwise => send the message to the second subscriber (check out serverless config)